pages/events/index.vue

```diff
@@ -251,6 +252,30 @@ export default class extends QiskitPage {
 
   &__filters-time {
     margin-top: $layout-03;
+    // deep selector idea taken from
```

Comment:
Maybe I should be using [this approach that already exists in the code base](https://github.com/Qiskit/qiskit.org/blob/master/components/learn/TheLearningResourceList.vue#L194), for consistency?

Reply:
Hey @y4izus, I created a refactor story here:

https://github.com/Qiskit/qiskit.org/issues/950

Reply:
Cool! thanks!